fli-iam/shanoir-ng
# Shanoir issue#1194 Add events / channels file to BIDS scans.tsv

How to test:
- Import EEG data
- Go to BIDS tree
- Check scans.tsv file for events.tsv and channel.tsv files

Closes #1194
